workflow-templates/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Link Markdown
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check_links:
+    name: Links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Actions Checkout
+        uses: actions/checkout@v2
+      - name: Check Links
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with: 
+          file-extension: '.md'
+          check-modified-files-only: 'yes'
+          config-file: '.github/workflows/markdown_links_config.json'
+
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.40
+          args: --timeout 5m0s
```

Comment:
Same as above, specify a category for what it should execute on.

For linting, you can indeed use the check-modified-files-only tag as linting only matters for modified files.

Reply:
Modified

Reply:
I looked at the golangci-lint documentation and it seems its for Go code linting, not for markdown linting. Unless I'm missing something? https://golangci-lint.run/

We want to accomplish Markdown linting as well as part of this workflow. Check out  https://github.com/marketplace/actions/markdown-linting-action or any other marketplace actions for Markdown linting